Sources/Features/Client/BacktraceReporter.swift

```diff
@@ -30,11 +31,17 @@ final class BacktraceReporter {
         self.repository = try PersistentRepository<BacktraceReport>(settings: dbSettings)
         let attributesProvider = AttributesProvider(reportHostName: dbSettings.reportHostName)
         self.attributesProvider = attributesProvider
-        self.backtraceOomWatcher = BacktraceOomWatcher(
-            repository: self.repository,
-            crashReporter: self.reporter,
-            attributes: attributesProvider,
-            backtraceApi: self.api)
+        if #available(iOS 15.3.1, *) {
+            BacktraceLogger.debug("The OOM support is disadbled for this version of iOS. Skipping OOM check.")
+            oomSupport = false
+        }
+        if(oomSupport == true){
```

Comment:
- if style seems to be without parentheses in this app ie `if self.reporter.oomSupport` vs `if(oomSupport == true)`
- also, the `== true` seems superfluous here
- could the `oomSupport` bool be named better? We now have `detectOom` and `oomSupport` which seems like the same property. Would it be better to force `self.configuration.detectOom` to be false instead of introduced another boolean?

Reply:
These have been implemented